cf-net/cf-net.c

```diff
@@ -472,7 +472,9 @@ static int CFNetRun(CFNetOptions *opts, char **args, char *hostnames)
     int ret = 0;
```

Comment:
So `ret` was not set to anything else than `0` before? Weird that we haven't been getting any warning/errors for this.

Reply:
I guess because we return `ret` in the end.